VDA-SBT/ChangeLog.md

```diff
@@ -1,2 +1,9 @@
 # Version 0.0.X
```

Comment:
Please copy format and name of our other CHANGELOG.md files for consistency.

See example: https://github.com/verida/verida-js/blob/main/packages/client-ts/CHANGELOG.md

Please change for all blockchain repos.

Reply:
Issue was lack of `.env` file.

@ITStar10 Please add instructions (and an example file) on how to configure `.env` file for all blockchain packages.